tests/integration/datadictwithobjid/schemas/dictionary.json

```diff
@@ -1 +1 @@
equences", "adapter_content", "kmer_content", "read_groups"], "namespace": "http://gdc.nci.nih.gov", "submittable": false, "project": "*", "program": "*", "additionalProperties": false, "type": "object"}, "keyword.yaml": {"description": "A keyword for a project.", "links": [{"backref": "keywords", "name": "projects", "multiplicity": "many_to_many", "required": true, "target_type": "project", "label": "describe"}], "uniqueKeys": [["id"], ["project_id", "submitter_id"]], "validators": null, "id": "keyword", "$schema": "http://json-schema.org/draft-04/schema#", "properties": {"updated_datetime": {"$ref": "_definitions.yaml#/datetime"}, "created_datetime": {"$ref": "_definitions.yaml#/datetime"}, "keyword_name": {"type": "string", "description": "The name of the keyword."}, "submitter_id": {"type": ["string", "null"]}, "state": {"$ref": "_definitions.yaml#/state"}, "project_id": {"type": "string"}, "type": {"enum": ["keyword"]}, "id": {"systemAlias": "node_id", "$ref": "_definitions.yaml#/UUID"}, "projects": {"$ref": "_definitions.yaml#/to_many"}}, "category": "administrative", "title": "Keyword", "systemProperties": ["id", "project_id", "state", "created_datetime", "updated_datetime"], "required": ["submitter_id", "projects"], "namespace": "http://gdc.nci.nih.gov", "submittable": true, "project": "*", "program": "*", "additionalProperties": false, "type": "object"}, "submitted_somatic_mutation.yaml": {"description": "Data file containing somatic mutation calls from a read group.\n", "links": [{"backref": "submitted_somatic_mutations", "name": "read_groups", "multiplicity": "many_to_many", "required": true, "target_type": "read_group", "label": "derived_from"}], "uniqueKeys": [["id"], ["project_id", "submitter_id"]], "validators": null, "id": "submitted_somatic_mutation", "$schema": "http://json-schema.org/draft-04/schema#", "properties": {"read_groups": {"$ref": "_definitions.yaml#/to_many"}, "data_type": {"term": {"$ref": "_terms.yaml#/data_type"}, "type": "string"}, "data_format": {"term": {"$ref": "_terms.yaml#/data_format"}, "type": "string"}, "data_category": {"term": {"$ref": "_terms.yaml#/data_category"}, "type": "string"}, "experimental_strategy": {"term": {"$ref": "_terms.yaml#/experimental_strategy"}, "type": "string"}, "type": {"enum": ["submitted_somatic_mutation"]}, "total_variants": {"type": "integer", "description": "The total number of variants detected carrying a base change difference from the reference genome."}, "$ref": "_definitions.yaml#/data_file_properties"}, "category": "data_file", "title": "Submitted Somatic Mutation", "systemProperties": ["id", "project_id", "created_datetime", "updated_datetime", "state", "file_state", "error_type"], "required": ["submitter_id", "file_name", "file_size", "data_format", "md5sum", "data_category", "data_type", "experimental_strategy", "read_groups"], "namespace": "http://gdc.nci.nih.gov", "submittable": true, "project": "*", "program": "*", "additionalProperties": false, "type": "object"}}
```

Comment:
Describing the case since the file is minimized: in the `Case` node, in addition to the existing `consent_codes` property, i added properties `consent_codes_ints`, `consent_codes_floats` and `consent_codes_bools`
```
"properties": {
    "consent_codes": {
        "type": "array",
        "description": "",
        "items": {
            "type": "string"
        }
    },
    "consent_codes_ints": {
        "type": "array",
        "description": "testing submission of array of ",
        "items": {
            "type": "integer"
        }
    },
    "consent_codes_floats": {
        "type": "array",
        "description": "",
        "items": {
            "type": "number"
        }
    },
    "consent_codes_bools": {
        "type": "array",
        "description": "",
        "items": {
            "type": "boolean"
        }
    },
```